### ISSUE TITLE
Can't find variable: canonicalizeIP

### ISSUE THREAD

**Issue by kahnpoint** [oven-sh/bun#3837]
### What version of Bun is running?

0.7.1

### What platform is your computer?

Linux 5.15.90.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

Hello everyone,

I am trying to use the postgres-js driver to connect to a database that requires SSL. 

When using bun run, this code works on non-SSL databases:
```javascript
import postgres from 'postgres';
const myClient = postgres({
    user: env.pg.user,
    password: env.pg.password,
    host: env.pg.host,
    database: env.pg.database,
    port: env.pg.port,
},
);
```
 But as soon as the ssl argument is added: 
```javascript
  ssl: {
    rejectUnauthorized: true,
    ca: fs.readFileSync("ca-certificate.crt").toString(),
  }
```

It throws :
```
ReferenceError: Can't find variable: canonicalizeIP at node:tls:79
```

The only reference to canonicalizeIP that I could find is here:
https://github.com/nodejs/node/blob/0da3f61a19c05508e437144a3a1dc5959a41d5d1/src/cares_wrap.cc#L1538

Is this just one of the unsupported nodejs calls? Is there anything I can do about it? Thank you for your time!

-adam

**Comment by Electroid** [comment]
Thanks for reporting, this is likely an issue with our `node:tls` implementation: https://github.com/oven-sh/bun/blob/0ea59b4c93e2fa285955e9129b3f219b07e30779/src/js/node/tls.js#L9

**Comment by Jarred-Sumner** [comment]
This has been fixed in Bun v0.7.1

**Comment by kahnpoint** [comment]
confirmed as working, thank you! 🙌
